giantswarm/cert-manager-app
# update to 0.15.2

towards giantswarm/giantswarm/issues/10403

## testing progress:

### new clusters

- [x] deploy to fresh nodepool aws cluster
   - [x] deploy helloworld manifest and get a cert for the ingress
- [x] deploy to fresh legacy aws cluster (`gauss/x7dh3`)
   - [x] deploy helloworld manifest and get a cert for the ingress
- [x]  deploy to fresh azure cluster (`godsmack/b87jc`)
   - [x] deploy helloworld manifest and get a cert for the ingress
- [x]  deploy to fresh kvm cluster (`gorgoth/rqmc6`)
   - [ ] deploy helloworld manifest and get a cert for the ingress

### cluster upgrades

test procedure for all providers:

1. deploy helloworld app and get a certificate for the ingress
2. upgrade the cluster to the new release
3. ensure the ingress tls config still works before/after re-deploying cert-manager

- [ ] prepare release branches
- [x] upgrade legacy aws cluster (optional app)
- [ ] upgrade aws nodepool cluster (pre-installed app)
- [ ] upgrade azure cluster (optional app)
- [ ] upgrade kvm cluster (optional app)
